src/ui/buttons/buttonDesigns/constants.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+export const DESIGN_SMALL_BUTTON_CONFIG = {
+    runOnce:                        false,
+    min:                            BUTTON_SIZE_STYLE.tiny.minWidth,
+    smallMax:                       BUTTON_SIZE_STYLE.small.maxWidth,
+    mediumMax:                      BUTTON_SIZE_STYLE.medium.maxWidth,
```

Comment:
Can this just be max instead of medium max

Reply:
done